### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Home Page] : Captions and transcription is not available while playing the pre-recorded video.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9031]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)

**Repro Steps:**
1. Open the above URL in the browser.
2. Home page will be appeared.
3. Press tab key and navigate to the 'Watch Video' button and press enter key to activate it.
4. Observe the issue.

**Actual Result:**
Captions and transcription options are not present for the pre-recorder video under Watch Video.

**Expected Result:**
Captions and transcription options should be available for the pre-recorder video under Watch Video.

**Note:**
Same issue is repro for the video player under Product Video and Media screens as there are no transcript options are present.

**User Experience:**
A person who can't listen to the voice after playing the video will find difficulty understanding voice if 'sub title' and Transcript' will not be available.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/07.01

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Captions or transcription is not available while playing the pre-recorded video](https://github.com/user-attachments/assets/f2207375-559b-4cee-a121-57d378551a4d)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Note: Need to add caption sources to https://github.com/microsoft/AdaptiveCards/blob/599d791a0ff0ece2214407e271e46401b10b91ce/source/nodejs/adaptivecards-site/themes/adaptivecards/layout/index.ejs#L3-L12

**Comment by anna-dingler** [comment]
We have released another version of the website, and this fix is available. Thanks!
